modules/core/src/main/java/org/apache/ignite/cache/query/VectorQuery.java

```diff
@@ -159,6 +165,27 @@ public int getK() {
         return k;
     }
 
+    /**
+     * Sets k (the number of vectors to return).
```

Comment:
The javadoc does not reflect the method.

Reply:
Fixed